.github/workflows/ci.yml

```
@@ -21,29 +21,169 @@ jobs:
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.12'
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -e ".[dev]"
+
+      - name: Run ${{ matrix.description }}
+        run: make ${{ matrix.make-target }}
+        env: ${{ matrix.env }}
```

actionlint (1.7.7)

type of expression at "env" must be object but found type string

(expression)